frontend/src/pages/Community.vue

```diff
@@ -0,0 +1,264 @@
+            currentPage: this.currentPage,
+            pageSize: this.pageSize,
+          })
+          this.verifyLogin()
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    deleteContribution(data) {
+      this.$apollo
+        .mutate({
+          fetchPolicy: 'no-cache',
+          mutation: deleteContribution,
+          variables: {
+            id: data.id,
+          },
+        })
+        .then((result) => {
+          this.toastSuccess(this.$t('contribution.deleted'))
+          this.updateListContributions({
+            currentPage: this.currentPage,
+            pageSize: this.pageSize,
+          })
+          this.verifyLogin()
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    updateListAllContributions(pagination) {
+      this.$apollo
+        .query({
+          fetchPolicy: 'no-cache',
+          query: listAllContributions,
+          variables: {
+            currentPage: pagination.currentPage,
+            pageSize: pagination.pageSize,
+          },
+        })
+        .then((result) => {
+          const {
+            data: { listAllContributions },
+          } = result
+          this.contributionCountAll = listAllContributions.contributionCount
+          this.itemsAll = listAllContributions.contributionList
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    updateListContributions(pagination) {
+      this.$apollo
+        .query({
+          fetchPolicy: 'no-cache',
+          query: listContributions,
+          variables: {
+            currentPage: pagination.currentPage,
+            pageSize: pagination.pageSize,
+          },
+        })
+        .then((result) => {
+          const {
+            data: { listContributions },
+          } = result
+          this.contributionCount = listContributions.contributionCount
+          this.items = listContributions.contributionList
+        })
+        .catch((err) => {
+          this.toastError(err.message)
+        })
+    },
+    verifyLogin() {
+      this.$apollo
+        .query({
+          query: verifyLogin,
+          fetchPolicy: 'network-only',
+        })
+        .then((result) => {
+          const {
+            data: { verifyLogin },
+          } = result
+          this.$store.dispatch('login', verifyLogin)
+        })
+        .catch(() => {
+          this.$emit('logout')
+        })
+    },
+    updateContributionForm(item) {
+      this.form.id = item.id
+      this.form.date = item.contributionDate
+      this.form.memo = item.memo
+      this.form.amount = item.amount
+      this.updateAmount = item.amount
+      this.tabIndex = 0
+    },
+    updateTransactions(pagination) {
+      this.$emit('update-transactions', pagination)
+    },
+  },
+  created() {
+    // verifyLogin is important at this point so that creation is updated if they are deleted in a session in the admin area.
```

Comment:
```suggestion
    // verifyLogin is important at this point so that creation is updated on reload if they are deleted in a session in the admin area.
```